test/IECoreImage/DisplayDriverServerTest.py

```diff
@@ -118,6 +119,47 @@ def testPortRangeRegistry( self ) :
 		s2 = IECoreImage.DisplayDriverServer()
 		self.assertEqual( s2.portNumber(), 45021 )
 
+	def testMergeMap( self ) :
```

Comment:
This approach to testing seems a little weak - there's nothing to show that the right data is sent successfully to the merged driver, or that the merged driver is closed at the right time. It also forces to expose the map publicly, which I think would be better left private. Can we beef this up by sending actual data and testing that it arrives in an ImageDisplayDriver appropriately?

Reply:
As much as I do agree that it is a little weak, I don't think it is the job of this test to make sure that the the image driver receives correct information since that is the job of the image writer test.

Here I'm only trying to make sure that the functionality of the merge map is adhered to. However, I do see your point that it's important to test that the image being sent over is closed correctly even if the code there hasn't changed.

I could similar tests as in `testTransfer()` in `ImageDisplayDriverTest.py` to make sure that the image is closed correctly and that data is being sent over.

Reply:
Also we have internal reasons at IE for exposing the map publicly

Reply:
What are the reasons?

Reply:
We wanted a way of checking which version of Cortex we're using. So querying if the function exists, but we could do something else.

Reply:
Thanks for removing - I don't think that was a strong enough reason to expose the internals.